pubsublite/internal/wire/rpc.go

```diff
@@ -130,7 +131,29 @@ func retryableReadOnlyCallOption() gax.CallOption {
 	})
 }
 
-const pubsubLiteDefaultEndpoint = "-pubsublite.googleapis.com:443"
+const (
+	pubsubLiteDefaultEndpoint = "-pubsublite.googleapis.com:443"
+	pubsubLiteDomain          = "pubsublite.googleapis.com"
```

Comment:
Nit; can you call this pubsubLiteErrorDomain? I was confused and thought this was an actual URL domain

Reply:
Done